packages/connection-ui/package.json

```
@@ -0,0 +1,39 @@
+{
+	"name": "jetpack-connection-manager-ui",
+	"version": "1.0.0",
+	"description": "Jetpack Connection Manager UI",
+	"main": "_inc/admin.jsx",
+	"repository": "https://github.com/Automattic/jetpack-connection-ui",
+	"author": "Automattic",
+	"license": "GPL-2.0-or-later",
+	"scripts": {
+		"build": "yarn install-if-deps-outdated && yarn clean && yarn build-client",
+		"build-client": "gulp",
+		"clean": "rm -rf build/",
+		"install-if-deps-outdated": "yarn check 2> /dev/null || yarn install --check-files --production=false --frozen-lockfile",
+		"watch": "yarn build && yarn gulp watch"
+	},
+	"dependencies": {
+		"@automattic/calypso-build": "6.5.0",
+		"@babel/core": "7.12.10",
+		"@babel/helper-module-imports": "7.12.5",
+		"@babel/preset-env": "7.12.11",
+		"@babel/register": "7.12.10",
+		"@wordpress/data": "4.26.1",
+		"fancy-log": "1.3.3",
+		"gulp": "4.0.2",
+		"react": "16.14.0",
+		"react-dom": "16.14.0",
+		"static-site-generator-webpack-plugin": "3.4.2",
+		"webpack": "4.45.0"
+	},
+	"devDependencies": {
+		"@wordpress/dependency-extraction-webpack-plugin": "2.9.0",
+		"enzyme": "^3.11.0",
+		"jest": "26.6.3"
+	},
+	"engines": {
+		"node": "^12.19.1",
+		"yarn": "^1.22.10"
```

Should we stick to dependencies similar to the one in the main repo, to avoid having to switch between node versions when working in one vs. the other?

```suggestion
		"node": "^12.18.0",
		"yarn": "^1.3.2"
```

---

That's a really good point, thanks!